lib/presentation/extensions/send_file_extension.dart

```diff
@@ -93,17 +93,15 @@ extension SendFileExtension on Room {
       }
 
       final formattedDateTime = DateTime.now().getFormattedCurrentDateTime();
-      final targetPath =
-          await File('${tempDir.path}/$formattedDateTime${fileInfo.fileName}')
-              .create();
-      fileInfo = fileInfo as ImageFileInfo;
+      final fileName = _generateThumbnailFileName(formattedDateTime, fileInfo);
+      final targetPath = await _getTargetParh(tempDir, fileName);
```

Comment:
it is a file, not a path. So, `_createThumbnailFile`

Reply:
done